content/doc/book/system-administration/administering-jenkins.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: section
+title: Administering Jenkins
+---
+
+== `JENKINS_HOME` directory
+Jenkins needs some disk space to perform builds and keep archives.
+You can check this location from the configuration screen of Jenkins.
+By default, this is set to `~/.jenkins`, but you can change this in one of the following ways:
+
+* Set the `JENKINS_HOME` environment variable to the new home directory before launching the servlet container.
+* Set the `JENKINS_HOME` system property in the servlet container.
+* Set the `JNDI` environment entry `JENKINS_HOME` to the new directory.
+See the link:/doc/book/installing/index[Installing Jenkins] for more about how to do this for your container.
+
+You can change this location after you've used Jenkins for a while, too.
+To do this, stop Jenkins completely, move the contents from the old `JENKINS_HOME` to the new home, set the new `JENKINS_HOME`, and restart Jenkins.
+`JENKINS_HOME` has a reasonably obvious directory structure that looks like the following:
+
+[width="100%",cols="100%",]
+|===
+a|
+....
+JENKINS_HOME
+    +- config.xml     (jenkins root configuration)
```

Comment:
For this and subsequent lines, do you want the descriptions in parentheses or maybe shown as comments?

Reply:
Need more detail.

Reply:
Descriptions in parentheses like this doc:
https://www.jenkins.io/doc/book/scaling/architecting-for-scale/#Anatomy%20of%20a%20$JENKINS_HOME

Reply:
I actually kind of like the parentheses myself but wanted to ask the question since I know people who feel strongly about using inline comment format.  So I will resolve this issue.